### ISSUE TITLE
Directory displays but doesn't CD into it

### ISSUE THREAD

**Issue by bytor99999** [wting/autojump#427]
I know I am just being stupid, but I followed the very simple install directions for Mac. brew install autojump. However, when I run it, all it does is return the path of the directory I wanted to cd into, it does not actually cd into it, so I have to select it, and then type cd space CMD-V to paste it. It works fine on my laptop at home, but not my work machine.

Thanks

**Comment by bendem** [comment]
you might want to take a look at all the reports linked in https://github.com/wting/autojump/issues/416

**Comment by wting** [comment]
Hey @bytor99999,

The Homebrew post install message should have included additional instructions on how to finish installing and using autojump. Please try re-installing autojump and following those instructions (unfortunately there's no easily linkable documentation at the moment).

**Comment by bytor99999** [comment]
That is exactly what I followed, and the same with the link bendem posted. That was the first place I looked when it wasn't working. The weird thing is that it only displays the directory and not cd into it.

I have added 

[[ -s /Users/me/.autojump/etc/profile.d/autojump.sh ]] && source /Users/me/.autojump/etc/profile.d/autojump.sh

autoload -U compinit && compinit -u

to my .zshrc and re sourced the file. But this is all I got was the directory displayed not cd'd into.

But I think the issue in my case and for people's future reference is a long time ago we needed to have an alias for autojump, so in my alias.zsh file I had a line

alias j="autojump"

Which when I just removed it, autojump started to work. And with just "j" and no alias needed.

Thanks

Mark
